cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -641,11 +641,12 @@ func sglogBranches(key string, branches []zoekt.RepositoryBranch) sglog.Field {
 }
 
 func (s *Server) indexArgs(opts IndexOptions) *indexArgs {
+	parallelism := s.CPUCount / s.IndexConcurrency
```

Comment:
minor: do you think we should round up rather than down? I would rather slightly oversubscribe and let the OS take care of it rather than under utilize. I suppose the issue is RAM? But even then my gut says obsersubscribe.

Reply:
Indeed I think that's a better trade-off, will update this.

I was thinking about it like... if you set CPU fraction then you want that to be an upper-bound on CPU use... but in practice it's more like a hint?